serverless/webpack.config.js

```diff
@@ -1,7 +1,36 @@
 const serverlessWebpack = require('serverless-webpack');
+const babelOptions = require('../babel.config');
+const path = require('path');
 
 module.exports = {
   entry: serverlessWebpack.lib.entries,
   target: 'node',
   mode: 'production',
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx|ts|tsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: require.resolve('babel-loader'),
+          options: {
+            ...babelOptions,
+          },
+        },
+      },
+    ],
+  },
+  optimization: {
+    // We do not want to minimize our code.
```

Comment:
Thinking about it, why, actually? Did the minimizer cause bugs for us? Or did it break source mapping?

Reply:
yeah, got the javascript heap our of memory error, at least on my local.

I'll try to give it a go, maybe in the CI it will pass although I would prefer to have a setup that doesn't crash on my local :D

Also I think by default Uglify is called to do the minification, ill try to give a different plugin a go too. It's been a known fact for a while that a smaller bundle has a positive performance impact on lambdas (albeit increases memory usage slightly, but we are not charged by that). See: https://medium.com/capital-one-tech/applying-minification-and-uglification-to-aws-lambda-functions-dbc7ad75241

Reply:
this is weird, this:
![image](https://user-images.githubusercontent.com/11645956/115207420-7e379100-a0f3-11eb-9ac8-b641fd36f207.png)
actually creates a bundle of 24MB, with uglification off it's only 17MB. Not sure if there is any point to explore if further as the default minification/uglification hangs anyway and this one is clearly worthless.